### PR TITLE
make this package installable by pip

### LINKED ISSUE #98: packaging - creating an installable version of the software

### PR DESCRIPTION
**Pull request by prjemian**
- close #98

### CODE REVIEW

**Comment by prjemian** [comment]
Took me a while to figure this one out: [4a3a140](https://github.com/BCDA-APS/gemviz/pull/152/commits/4a3a1407b389f5ba241bd22983dae5e0bef5d754)

Seems that the "header" file (named when promoting a widget in the designer) provides the indication for PyQt5 to locate the source code.  I'm guessing it strips the `.h` to find the Python module to be loaded with the supporting Python code.  So, that "header" file name must include the Python package.module to find the Python code.

**Comment by prjemian** [comment]
When testing on Windows, notice that Python files are sometimes autocreated from the `.ui` files.  Need to ignore them.

**Comment by prjemian** [comment]
With these packaging changes, a _starter_ is needed to run the application in debug mode. `dev_starter.py`:

```py
from gemviz import app as gemviz_app

gemviz_app.main()
```

File: gemviz/aboutdialog.py [hunk 1]
```diff
@@ -59,24 +67,25 @@ def doUrl(self, url):
...
+        from .licensedialog import LicenseDialog
+
         self.setStatus("opening License in new window")
-        license=LicenseDialog(self)
+
+        license = LicenseDialog(self)
         license.finished.connect(self.clearStatus)
-        license.open()   
-    
+        license.open()  # modal: must close licensedialog BEFORE aboutdialog
```
Comment (rodolakis) [inline gemviz/aboutdialog.py:85]: I liked it better when the `about` dialog closes when I open the `license`; I find it quite annoying to have to close `license` THEN `about`. But that is not a hill I am willing to die on.

**Review by rodolakis — APPROVED** [review]

**Pull request merged**